doc/tutorial/invoking-zeekctl.rst

```diff
@@ -1,9 +1,257 @@
 .. _invoking-zeekctl:
 
-================
-Invoking Zeekctl
-================
+.. _zeekcontrol cron: https://github.com/zeek/zeekctl?tab=readme-ov-file#zeekcontrol-cron-command
+
+.. _zeekcontrol documentation: https://github.com/zeek/zeekctl
+
+#############
+ ZeekControl
+#############
+
+ZeekControl (or ``zeekctl``) is Zeek's primary tool for cluster
+orchestration. In this case, a Zeek cluster is simply multiple Zeek
```

Comment:
I wonder if we could go fancy/abstract, but ignore if that seems to be too abstract.
```suggestion
orchestration. Technically, a Zeek cluster is a set of independent Zeek processes that analyze network traffic.
```